tests/sls/homu/valid_builders.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os.path
+import subprocess
+import toml
+
+from tests.util import Failure, Success
+
+
+def are_builders_valid(buildbot, homu_cfg, buildertype):
```

Comment:
Inline `are_builders_valid` into `run()`, which allows removing a bunch of duplicated error checking.